### PR TITLE
User story/2138 allow sectionstate to select a list of sections

### PR DESCRIPTION
**Pull request by randy-seng**
OP#2138
OP#2139

### CODE REVIEW

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_flows.py [hunk 1]
```diff
@@ -69,6 +66,21 @@ def _place_widgets(self) -> None:
     def action_buttons(self) -> list[CTkButton]:
         return self._action_buttons
 
+    def set_selection(self, flow_ids: list[str]) -> None:
+        pass
```
Comment (briemla) [inline OTAnalytics/plugin_ui/customtkinter_gui/frame_flows.py:70]: In `frame_sections.py` a `NotImplementedError` is raised. Should it be the same behaviour? Or should we remove the method completely?

**Review by briemla — COMMENTED** [review]
Looks good. Only a small question.

**Review by briemla — COMMENTED** [review]
Looks good. Only a small question.

File: OTAnalytics/plugin_ui/customtkinter_gui/frame_flows.py (hunk 1 above)
Comment (randy-seng) [reply OTAnalytics/plugin_ui/customtkinter_gui/frame_flows.py:70]: Yes, it is not being used so we can remove method completely.

**Review by briemla — APPROVED** [review]

**Pull request merged**